engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/UniverseSetupScreen.java

```diff
@@ -174,6 +177,7 @@ public String get() {
             @Override
             public void set(String value) {
                 selectedWorld = value;
+                camelLocation = findIndex(worlds,selectedWorld);
```

Comment:
Trivial code style issue with whitespace. Please be sure to set up and run Checkstyle on any files you edit to catch those warnings :-)